### PR TITLE
Terminal output

### LINKED ISSUE #1: implement NetworkTerminalOutput to server
create a output class that works for client - server socket communications

### LINKED ISSUE #4: fix the check for no content recieved on NetworkTerminalOutput
when the client just doesn't type any data and presses send, the client enters a loop.
the current check on NetworkTerminalOutput isn't doing the trick.

### PR DESCRIPTION
**Pull request by arielofer**
adding NetworkTerminalOutput class to allow printing the scene data to the client

fix #1, #4

### CODE REVIEW

File: output.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Output(object):
```
Comment (agamm) [inline output.py]: Why not include the library?

**Review by agamm — CHANGES_REQUESTED** [review]

File: output.py (hunk 1 above)
Comment (arielofer) [reply output.py]: fixed

File: network_terminal_output.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from gamebook.output import Output
+
+
+class NetworkTerminalOutput(Output):
+    def __init__(self, client):
+        self.client = client
+
+    def output(self, content):
+        if content == "":
+            self.client.send(b"no content recieved")
```
Comment (agamm) [inline network_terminal_output.py]: why `b"..."`?

File: server.py [hunk 3]
```diff
@@ -9,11 +10,9 @@
     socket.listen()
 
     client, address = socket.accept()
+    output_instance = NetworkTerminalOutput(client)
```
Comment (agamm) [inline server.py]: Can you use the library? so this will be encapsulated in the constructor?

**Review by agamm — CHANGES_REQUESTED** [review]

File: server.py (hunk 3 above)
Comment (arielofer) [reply server.py]: yeah i can use it, successfully downloaded it with pip install

Comment (agamm) [reply server.py]: Awesome. Add it an ask for a re-review.

File: server.py [hunk 4]
```diff
@@ -1,19 +1,28 @@
...
-    client, address = socket.accept()
-    with client:
-        print(f"connection made from {address}")
-        while True:
-            data = client.recv(1024)
-            if not data:
-                break
-            client.send(data)
-
+    socket.listen(5)
```
Comment (agamm) [inline server.py]: why 5?

File: server.py [hunk 5]
```diff
@@ -1,19 +1,28 @@
...
+            client.settimeout(15)
+            while True:
+                data = client.recv(1024)
+                server_output.output(f"recieved {data.decode('utf-8')}")
+                if data.decode('utf-8') == "quit":
+                    server_output.output(f"closing connection with {address}")
+                    client.close()
+                    break
+                else:
+                    client_output.output(data)
```
Comment (agamm) [inline server.py:28]: I assume this will change to game book?

**Review by agamm — APPROVED** [review]

File: server.py (hunk 4 above)
Comment (arielofer) [reply server.py]: forgot to remove it after i finished experimenting

File: server.py (hunk 5 above)
Comment (arielofer) [reply server.py:28]: yes

**Pull request merged**